SDK/Examples/CppDynamicNew/CMakeLists.txt

```diff
@@ -0,0 +1,75 @@
+#[[++
+
+Copyright (C) 2019 3MF Consortium (Original Author)
+
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
+
+1. Redistributions of source code must retain the above copyright notice, this
+list of conditions and the following disclaimer.
+2. Redistributions in binary form must reproduce the above copyright notice,
+this list of conditions and the following disclaimer in the documentation
+and/or other materials provided with the distribution.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR
+ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+This file has been generated by the Automatic Component Toolkit (ACT) version 1.5.0-develop3.
+
+Abstract: This is an autogenerated CMake Project that demonstrates the
+ usage of the Dynamic C++ bindings of the 3MF Library
+
+Interface version: 2.2.0
+
+
+]]
+
+cmake_minimum_required(VERSION 3.5)
+
+project(Example_ExtractInfo)
+set(CMAKE_CXX_STANDARD 11)
+
+# Determine the platform and set lib3mf_DIR accordingly
+if(WIN32)
+	# Path for Windows
+	set(lib3mf_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../lib3mf-2.3.1-Windows/lib/cmake/lib3mf")
```

Comment:
What happens when lib3mf_DIR is empty? Does this break the build?

I suppose we need to download the sdk from artifact and copy it here. Correct me if I am wrong.

Reply:
Then can we move it to CPackExamples? Since these works only with CPacked SDKs, for github actions, we can use the same.